staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4150,7 +4150,7 @@ type ServiceSpec struct {
 	// externalName is the external reference that discovery mechanisms will
 	// return as an alias for this service (e.g. a DNS CNAME record). No
 	// proxying will be involved.  Must be a lowercase RFC-1123 hostname
-	// (https://tools.ietf.org/html/rfc1123) and requires Type to be
+	// (https://tools.ietf.org/html/rfc1123) and requires `type` to be "ExternalName".
```

Comment:
I couldn't´t find any other place that uses backticks for `tyoe`

Reply:
I added backticks per this suggestion:

https://github.com/kubernetes/kubernetes/pull/97896#discussion_r554985889

Indeed, I can't find many usages of backticks in this file, but I didn't know if using backticks to annotate field names was a new convention.

Reply:
ups sorry, I missed it, my fault